wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/DirectedAcyclicGraph.kt

```diff
@@ -15,6 +15,8 @@
  */
 package com.squareup.wire.schema
 
+typealias Queue<T> = ArrayDeque<T>
```

Comment:
This puts the name into the public API which seems unnecessary. It's only used in one place, why not reference the type directly?

Reply:
will remove! It was just testing some stuff in the beginning and this was left over.

Reply:
fixed